packages/flutter/test/widget/flex_test.dart

```diff
@@ -48,10 +48,10 @@ void main() {
     expect(didReceiveTap, isTrue);
   });
 
-  testWidgets('Row and FlexJustifyContent.collapse', (WidgetTester tester) async {
+  testWidgets('Row and MainAxisSpace.min', (WidgetTester tester) async {
```

Comment:
Can you add the following tests:

 * `MainAxisSpace.min` combined with children that are bigger than the min result in a widget size that's between min and max.
 * The same as above, but with enough children to overflow max to show that the size is capped at max.
 * `MainAxisSpace.min` combined with `spaceBetween` achieves the effect that motivated you to add this feature in the first place (i.e., that if the min is small you get collapsing but if the min is large you get the free space in the middle).